SumOfTheDigits/src/com/company/Main.java

```diff
@@ -0,0 +1,9 @@
+package com.company;
```

Comment:
Why you need this file? You already have implemented main class in this project.